### ISSUE TITLE
cannot see the second letter

### ISSUE THREAD

**Issue by rossamurphy** [DavidLGoldberg/jumpy2#10]
Hi there, 
thanks for making this extension! 
there is an issue that it is very difficult to see the second letter of the label. Can you please make the label sizes bigger?
i.e. in the screenshot, it's difficult to figure out which letter comes after the b ....

<img width="1037" alt="image" src="https://user-images.githubusercontent.com/43931816/206303621-6bc6b9c3-48cf-4ee3-bc3d-f8b91ee14479.png">

**Comment by DavidLGoldberg** [comment]
Hi.  Sorry, I actually missed these until now.  My bad.

So, yeah, mine are not that skewed towards the right as in these images.  But it is slightly on mine too.  I wonder if something has changed.  Can you provide which theme, font, settings (if custom) you are using?  I'll do some experimentation.

There is another issue with linter warnings / errors that sometimes gets in my way too, I'd like to fix.

**Comment by danulqua** [comment]
@DavidLGoldberg My main setup is MonoLisa font and font-size 14px: 
![CleanShot 2023-02-07 at 08 53 33@2x](https://user-images.githubusercontent.com/38916225/217170366-32c3a848-edba-415e-bac7-9395b94e7eff.jpg)

MonoLisa font, but font-size 13px: 
![CleanShot 2023-02-07 at 08 54 57@2x](https://user-images.githubusercontent.com/38916225/217170424-d523e9fd-86b9-4079-99bf-103e6fd8b547.jpg)

monospace font, but font-size 13px: ![CleanShot 2023-02-07 at 08 55 20@2x](https://user-images.githubusercontent.com/38916225/217170514-dc89501d-ce6c-4e23-8dfa-dc49911027c9.jpg)

In conclusion, offset above the background depends on the font-size, it's inconsistent.

**Comment by DavidLGoldberg** [comment]
Had to do some cleanup.  Build etc.  I'll try to address this soon!

**Comment by rossamurphy** [comment]
thanks!

**Comment by DavidLGoldberg** [comment]
*font sizes are fixed in the new version!
*added a border.  See below if interested...

Working with the decorations is sort of miserable in VS Code if I'm being honest.

Maybe I/we can iterate on the look and feel if it's not great.  But overall I think it's a big step up.  Actually accomplished a few things I was looking to do.  I prefer smaller square labels.  It also prints better on selected text ranges etc now.

I Added a default border as you requested.  It's very subtle, but you can change it in the theme.  The border is what draws the lines between contiguous labels.  Unfortunately using default decorations merge contiguous ranges into one giant rectangle/oval (with radius) I can take a snapshot of a branch I have if really curious, but basically prohibits the borders between the labels (very weird).  So I have to do it on the :after {  } (where the :content lives).  

So there were a few reasons I couldn't / didn't want to use the more standard way, but I think this came out pretty decent.

**Comment by DavidLGoldberg** [comment]
@rossamurphy @danulqua This work out for you ok?

**Comment by danulqua** [comment]
@DavidLGoldberg in some way it got better, and in some way it got worse.

![CleanShot 2023-02-28 at 10 59 24@2x](https://user-images.githubusercontent.com/38916225/221803848-b9f2a0fd-cfd5-4a55-83cb-19612083d529.jpg)

Better: letters now fit the width of the green block, but not perfectly, see the `f` on line 72 and `m` on line 74. I am sure that this block needs some padding right.

Worse: if you compare previous screenshots with this one, you'll notice that green block's height was decreased and letters touch the bottom edge. Still a bit hard to recognize letters like `g`, `j`, `y` etc.

Leaving my setup again: font `MonoLisa`, size `14px`, line height `0`.
